tests/integration/test_playbook.yml

```diff
@@ -0,0 +1,7 @@
+---
+
+- name: Playbook for testing
```

Comment:
I think this is leftover from tests? Because we execute it from a different place

Reply:
True, accidentally added this to the commit. Fixed with a force-push